docs/ru/getting-started/tutorial.md

```diff
@@ -564,38 +564,38 @@ Example config for a cluster with three shards, one replica each:
 
 ``` sql
 CREATE TABLE tutorial.hits_local (...) ENGINE = MergeTree() ...
 ```
 
-Creating a distributed table providing a view into local tables of the cluster:
+Создадим распределённую таблицу, обеспечивающую представление над локальными таблицами кластера:
 
 ``` sql
 CREATE TABLE tutorial.hits_all AS tutorial.hits_local
 ENGINE = Distributed(perftest_3shards_1replicas, tutorial, hits_local, rand());
 ```
 
-A common practice is to create similar Distributed tables on all machines of the cluster. It allows running distributed queries on any machine of the cluster. Also there’s an alternative option to create temporary distributed table for a given SELECT query using [remote](../sql-reference/table-functions/remote.md) table function.
+Стандартная практика — создание одинаковых распределённых таблиц на всех узлах кластера. Это позволит запускать распределённые запросы с любого узла. Альтернативой может быть создание временной распределённой таблицы для заданного отдельно взятого запроса с использованием табличной функции [remote](../sql-reference/table-functions/remote.md).
 
-Let’s run [INSERT SELECT](../sql-reference/statements/insert-into.md) into the Distributed table to spread the table to multiple servers.
+Выполним [INSERT SELECT](../sql-reference/statements/insert-into.md) в распределённую таблицу, чтобы распределить данные по нескольким узлам.
 
 ``` sql
 INSERT INTO tutorial.hits_all SELECT * FROM tutorial.hits_v1;
 ```
 
-:::danger Notice
-This approach is not suitable for the sharding of large tables. There’s a separate tool [clickhouse-copier](../operations/utilities/clickhouse-copier.md) that can re-shard arbitrary large tables.
+:::danger Внимание!
+Этот подход не годится для сегментирования больших таблиц. Есть инструмент [clickhouse-copier](../operations/utilities/clickhouse-copier.md), специально предназначенный для перераспределения любых больших таблиц.
 :::
 
-As you could expect, computationally heavy queries run N times faster if they utilize 3 servers instead of one.
+Как и следовало ожидать, вычислительно сложные запросы работают втрое быстрее, если они выполняются на трёх серверах, а не на одном.
 
-In this case, we have used a cluster with 3 shards, and each contains a single replica.
+В данном случае мы использовали кластер из трёх сегментов с одной репликой для каждого.
 
-To provide resilience in a production environment, we recommend that each shard should contain 2-3 replicas spread between multiple availability zones or datacenters (or at least racks). Note that ClickHouse supports an unlimited number of replicas.
+В продуктивных окружениях для обеспечения надёжности мы рекомендуем чтобы каждый сегмент был защищён 2—3 репликами, разнесёнными на разные зоны отказоустойчивости или разные центры обработки данных (или хотя бы разные стойки). Особо отметим, что ClickHouse поддерживает неограниченное количество реплик.
```

Comment:
```suggestion
В производственных окружениях для обеспечения надёжности мы рекомендуем чтобы каждый сегмент был защищён 2—3 репликами, разнесёнными в разные зоны отказоустойчивости или разные центры обработки данных (или хотя бы разные стойки). Особо отметим, что ClickHouse поддерживает неограниченное количество реплик.
```

Reply:
> 

I guess ‘productive’ is more smoothly translated as calque “продуктивный”, because ‘промышленный’ in Russian means ‘industrial‘ (and not ‘productive’) in 99% of contexts.

Reply:
[Sorry, I meant ‘производственный’ as the same of ’промышленный’, it‘s also means industrial-related in most of Russian contexts]

Reply:
In original it's _production environment_ and refers to environment used to serve business processes, isn't it?
Not relative to _productive_

Reply:
‘Производственный‘ in Russian also tightly relates to [industrial] production, i. e. ability to produce large amounts of goods… ’В эксплуатационных окружениях…’?